nested/azuresearch-none.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+    ],
+    "variables": {
+        "documentation01": "This sub-template drives the Azure Search which is used as the access-point for other moodle VM's ",
+        "documentation02": "It expects certain values in the 'common' datastructure.",
+        "documentation03": "    azureSearchName                 - the name of the Azure Search.",
+        "documentation04": "    azureSearchSku                  - the level of Azure Search service.",
+        "documentation06": "    azureSearchReplicaCount         - number of Azure Search replicas.",
+        "documentation07": "    azureSearchPartitionCount       - number of Azure Search partitions.",
+        "documentation08": "    azureSearchHostingMode          - the type of Azure Search hosting mode.",
+        "azureSearchServiceId": "[resourceId('Microsoft.Search/searchServices', parameters('moodleCommon').azureSearchName)]"
+    },
+    "outputs": {
+        "azureSearchKey": {
+            "value": "[listAdminKeys(variables('azureSearchServiceId'), '2015-08-19').PrimaryKey]",
```

Comment:
Likewise, I'm pretty sure this line will fail when azureSearchSwitch is false. This should be similarly a hard-coded value like in redis-none.json.